node/task.ts

```diff
@@ -228,6 +228,24 @@ export function getInput(name: string, required?: boolean): string | undefined {
     return inval;
 }
 
+/**
+ * Gets the value of an input.
+ * If the value is not set, it will throw.
+ *
+ * @param     name     name of the input to get
+ * @returns   string
+ */
+export function getInputRequired(name: string): string {
+    var inval = im._vault.retrieveSecret('INPUT_' + im._getVariableKey(name));
```

Comment:
Please use the call of existing methods instead (like getInput(name, required) - to avoid duplication

Reply:
Do you mean that I should have `getInputRequired` call `getInput` so that the internals of that method are not duplicated?

Reply:
@Banner-Keith yep correct 👍